File: 3110-score-of-a-string/3110-score-of-a-string.cs
public class Solution {
    public int ScoreOfString(string s) {
        int result=0;
    for(int i=0;i<s.Length-1;i++){
        int current=(int)s[i];
        int next=(int)s[i+1];
        result+=Math.Abs(current-next);
    }
    return result;
    }
}

File: 3110-score-of-a-string/3110-score-of-a-string.js
/**
 * @param {string} s
 * @return {number}
 */
var scoreOfString = function(s) {
    let result=0;
    for(let i=0;i<s.length-1;i++){
        let current=s[i].charCodeAt(0);
        let next=s[i+1].charCodeAt(0);
        result+=Math.abs(current-next)
    }
    return result;
};